frontend/src/store/modules/place/actions.js

```diff
@@ -2,22 +2,28 @@ import httpService from '@/services/common/httpService';
 
 export default {
     loadCurrentPlace: ({ state, commit }, id) => {
-        return httpService.get('/places/' + id)
-            .then(function (response) {
-                commit('SET_CURRENT_PLACE', response.data.data);
-            })
-            .catch(function (err) {
-                console.log(err);
-            });
+        return new Promise((resolve, reject) => {
+            httpService.get('/places/' + id)
+                .then(function (response) {
+                    commit('SET_PLACE', response.data.data);
```

Comment:
No need for `commit('SET_PLACE', response.data.data);` here.  this state item unlikely will be reused in other components. You can do `this.place = response;` in component